From: Uwe Kleine-König <u.kleine-koenig@pengutronix.de>
Subject: Don't reference non-standard realtime group

There is no dedicated realtime group in most distributions. So make the
error message a bit more understandable for people not running Redhat
MRG.

Signed-off-by: Uwe Kleine-König <u.kleine-koenig@pengutronix.de>
Forwarded: id:1310590111-11484-1-git-send-email-u.kleine-koenig@pengutronix.de
Closes: http://bugs.debian.org/619938
---
 src/lib/rt-utils.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/src/lib/rt-utils.c
+++ b/src/lib/rt-utils.c
@@ -265,7 +265,8 @@
 	param.sched_priority = 1;
 	if (sched_setscheduler(0, SCHED_FIFO, &param)) {
 		fprintf(stderr, "Unable to change scheduling policy!\n");
-		fprintf(stderr, "either run as root or join realtime group\n");
+		fprintf(stderr, "Probably missing capabilities, either run as "
+				"root or increase RLIMIT_RTPRIO limits.\n");
 		return 1;
 	}
 
